Index: .idea/phpspec.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"PHPSpec\">\r\n    <suites>\r\n      <PhpSpecSuiteConfiguration>\r\n        <option name=\"myPath\" value=\"$PROJECT_DIR$\" />\r\n      </PhpSpecSuiteConfiguration>\r\n      <PhpSpecSuiteConfiguration>\r\n        <option name=\"myPath\" value=\"$PROJECT_DIR$/RegistroVital\" />\r\n      </PhpSpecSuiteConfiguration>\r\n      <PhpSpecSuiteConfiguration>\r\n        <option name=\"myPath\" value=\"$PROJECT_DIR$/RegistroVital\" />\r\n      </PhpSpecSuiteConfiguration>\r\n    </suites>\r\n  </component>\r\n</project>
===================================================================
diff --git a/.idea/phpspec.xml b/.idea/phpspec.xml
--- a/.idea/phpspec.xml	
+++ b/.idea/phpspec.xml	
@@ -11,6 +11,9 @@
       <PhpSpecSuiteConfiguration>
         <option name="myPath" value="$PROJECT_DIR$/RegistroVital" />
       </PhpSpecSuiteConfiguration>
+      <PhpSpecSuiteConfiguration>
+        <option name="myPath" value="$PROJECT_DIR$/RegistroVital" />
+      </PhpSpecSuiteConfiguration>
     </suites>
   </component>
 </project>
\ No newline at end of file
